Bottom: 181766c908905a728c390f1d26be4978c5bfd99f
Top:    30d69e7ee7cfa4b3321558975c4a35cf95283711
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 16:31:27 +0000

Remove left over debug


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index cfd5d6a..656cc68 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1220,8 +1220,6 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
 #endif
                               copper_appearance,
                               NULL));
-
-    //break;
   }
